lib/smart_proxy_ansible/runner/ansible_runner.rb

```diff
@@ -28,6 +28,7 @@ def initialize(input, suspended_action:, id: nil)
         @passphrase = action_input['secrets']['key_passphrase']
         @execution_timeout_interval = action_input[:execution_timeout_interval]
         @cleanup_working_dirs = action_input.fetch(:cleanup_working_dirs, true)
+        prune_known_hosts_on_first_execution(input.values)
```

Comment:
Would it be possible to use the reassembled ansible inventory as stored in `@inventory` instead of pulling things out of the raw input? Not a hard requirement though

Reply:
I chose to use `input.values` instead of `@inventory` because the latter doesn't include the `first_execution` parameter.
While extracting this value from `input.values` for each host adds a bit of overhead, it seems necessary in order to access `first_execution`.
Do you think there’s a cleaner or more efficient way to handle this?

Reply:
Does this mean that other parameters used in `prune_known_hosts_on_first_execution` are available through `@inventory`? If so, and just one `first_execution` param is missing, wouldn't it be logical to include that parameter there instead of rebuilding kind of the same information?

Reply:
Yes, the other parameters used in `prune_known_hosts_on_first_execution` are available through `@inventory`. However, I’m not sure I fully follow, I’m not really rebuilding anything. I’m simply using the original `input.values` because `@inventory` doesn’t include the `first_execution` field for each host.